### PR TITLE
added air gap step

### PR DESCRIPTION
**Pull request by dprodanov4**

### CODE REVIEW

File: asciidoc/components/longhorn.adoc [hunk 1]
```diff
@@ -274,3 +274,80 @@ defaultSettings:
 ----
 
 * After the image is build you can use it to install your OS and Longhorn will be deployed on top of RKE2.
+
+== Edge Image Builder air gap installation of Longhorn
```
Comment (atanasdinov) [inline asciidoc/components/longhorn.adoc:278]: Can we keep the consistency with the section above? One could be `Use Edge Image Builder to deploy Longhorn` and the other `Use Edge Image Builder to deploy Longhorn in an air gapped environment` or something similar.

File: asciidoc/components/longhorn.adoc [hunk 2]
```diff
@@ -274,3 +274,80 @@ defaultSettings:
...
+
+* We can install air gaped Longhorn using Edge Image Builder by providing the images to the configuration file like this:
```
Comment (atanasdinov) [inline asciidoc/components/longhorn.adoc:280]:
```suggestion
* We can install air gapped Longhorn using Edge Image Builder by providing the images to the configuration file like this:
```

File: asciidoc/components/longhorn.adoc [hunk 3]
```diff
@@ -274,3 +274,80 @@ defaultSettings:
...
+apiVersion: 1.0
+image:
+    imageType: raw
+    arch: x86_64
+    baseImage: SLE-Micro.x86_64-5.5.0-Default-GM.raw
+    outputImageName: longhorn.raw
+kubernetes:
+  version: v1.28.7+rke2r1
+  network:
+    apiVIP: <ip>
```
Comment (atanasdinov) [inline asciidoc/components/longhorn.adoc:316]: We can remove this.

File: asciidoc/components/longhorn.adoc (hunk 1 above)
Comment (dprodanov4) [reply asciidoc/components/longhorn.adoc:278]: Will update now

**Review by atanasdinov — APPROVED** [review]

**Pull request merged**